config/level/php/php70.yml

```diff
@@ -11,3 +11,4 @@ services:
     Rector\Php\Rector\List_\ListSwapArrayOrderRector: ~
 
     Rector\Php\Rector\FuncCall\CallUserFuncRector: ~
+    Rector\Php\Rector\FuncCall\EregToPregMatchRector: ~
```

Comment:
Add current rector too
Maybe validation that package rectors are in at one set?